schemas/test/en/test_list_collector_repeating_blocks_section_summary.json

```diff
@@ -0,0 +1,361 @@
+                                    "id": "add-question-companies",
+                                    "type": "General",
+                                    "title": "What is the name and registration number of the company?",
+                                    "answers": [
+                                        {
+                                            "id": "company-or-branch-name",
+                                            "label": "Name of UK company or branch (Mandatory)",
+                                            "mandatory": true,
+                                            "type": "TextField"
+                                        }
+                                    ]
+                                }
+                            },
+                            "repeating_blocks": [
+                                {
+                                    "id": "companies-repeating-block-1",
+                                    "type": "ListRepeatingBlock",
+                                    "question": {
+                                        "id": "companies-repeating-block-1-question",
+                                        "type": "General",
+                                        "title": {
+                                            "text": "Give details about {company_name}",
+                                            "placeholders": [
+                                                {
+                                                    "placeholder": "company_name",
+                                                    "value": {
+                                                        "source": "answers",
+                                                        "identifier": "company-or-branch-name"
+                                                    }
+                                                }
+                                            ]
+                                        },
+                                        "answers": [
+                                            {
+                                                "id": "registration-number",
+                                                "label": "Registration number (Mandatory)",
+                                                "mandatory": true,
+                                                "type": "Number",
+                                                "maximum": {
+                                                    "value": 999,
+                                                    "exclusive": false
+                                                },
+                                                "decimal_places": 0
+                                            },
+                                            {
+                                                "id": "registration-date",
+                                                "label": "Date of Registration (Mandatory)",
```

Comment:
If you enter a date in the future you'll then be asked for details about how your company has been trading over the last -124 years. 😁 Not an issue with the feature but might be nice to cap the date

Reply:
Hahah yeah noticed this in the PR run through, will add validation for this (assumign it works 🤞 ).
Regarding this causing a dependancy on a repeating block on a prev repeating block, the requirements do not specify that must be handled at this moment in time.